WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -101,6 +102,7 @@ def __init__( self, jobID = None, jobReport = None ):
     self.sandboxSizeLimit = gConfig.getValue( self.section + '/OutputSandboxLimit', 1024 * 1024 * 10 )
     self.cleanUpFlag = gConfig.getValue( self.section + '/CleanUpFlag', True )
     self.pilotRef = gConfig.getValue( '/LocalSite/PilotReference', 'Unknown' )
+    self.boincUserID = gConfig.getValue( '/LocalSite/BoincUserID', '' )
```

Comment:
Just a naive question: doesn't ```0``` make more sense as a default value for a ID ? It is really just a detail